packages/back-end/src/api/ingestion/ingestion.router.ts

```diff
@@ -0,0 +1,63 @@
+import { z } from "zod";
+import { Router } from "express";
+import {
+  createApiRequestHandler,
+  validateIsSuperUserRequest,
+} from "back-end/src/util/handler";
+import { SDKConnectionInterface } from "back-end/types/sdk-connection";
+import { findSdkConnectionsAcrossMultipleOrgs } from "back-end/src/models/SdkConnectionModel";
+import { _dangerousGetAllGrowthbookClickhouseDataSources } from "back-end/src/models/DataSourceModel";
+
+interface SdkInfo {
+  organization: string;
+  client_key: string;
+  datasource: string;
+}
+
+interface GetDataEnrichmentResponse {
+  sdkData: {
+    [key: string]: SdkInfo;
+  };
+}
+
+function sdkInfo(conn: SDKConnectionInterface, datasource: string): SdkInfo {
+  return {
+    organization: conn.organization,
+    client_key: conn.id,
+    datasource,
+  };
+}
+
+export const getDataEnrichment = createApiRequestHandler({
+  bodySchema: z.never(),
+  querySchema: z.never(),
+  paramsSchema: z.never(),
+})(
+  async (req): Promise<GetDataEnrichmentResponse> => {
+    // Must be a super-user to make cross-org mongo queries
+    await validateIsSuperUserRequest(req);
+
+    const dataSources = await _dangerousGetAllGrowthbookClickhouseDataSources();
+    const dataSourcesByOrgId = Object.fromEntries(
+      dataSources.map((ds) => [ds.organization, ds.id])
+    );
+    const sdkConnections = await findSdkConnectionsAcrossMultipleOrgs(
+      Object.keys(dataSourcesByOrgId)
+    );
+    const sdkData = Object.fromEntries(
+      sdkConnections.map((conn) => [
+        conn.id,
+        sdkInfo(conn, dataSourcesByOrgId[conn.organization] || ""),
```

Comment:
Instead of defaulting to an empty string, we should skip these entries and log an error

Reply:
I realized that with some of the changes we made this isn't actually necessary anymore. The query that pulls these `conn`s is by their organization which is just the keys of the object being indexed here, so this should actually be safe now.